Importer/Log/CsvLog.php

```diff
@@ -50,9 +50,12 @@ public function addDataError($index, $errorMessage)
     /**
      * {@inheritdoc}
      */
-    public function addCriticalError($errorMessage)
+    public function addCriticalError($message, $userError = false)
```

Comment:
So on the case of an application error that is thrown because it's Tuesday and we should never import on Tuesday's, what's the value of `errorReason` and what's the value of `userMessage`?